packages/common/spec-guide/index.js

```diff
@@ -0,0 +1,17 @@
+const { asyncRoute } = require('@parameter1/base-cms-utils');
+const fetch = require('node-fetch');
+
+const { SPEC_GUIDE_DOCS_API_KEY } = process.env;
```

Comment:
This should pull from envalid, not the process env directly.
```suggestion
const fetch = require('node-fetch');
const { SPEC_GUIDE_DOCS_API_KEY } = require('../env');

```